packages/main/src/RadioButton.js

```diff
@@ -182,12 +182,12 @@ const metadata = {
 		},
 
 		/**
-		 * Defines the accessible name of the component.
+		 * Defines the accessible aria name of the component.
```

Comment:
* Defines the accessible ARIA name of the component.

Reply:
All other components use lowercase "aria". I can make separate pull request to improve this